server/conf/application.test.conf

```diff
@@ -16,6 +16,10 @@ play.evolutions.db.default.enabled = "true"
 azure.blob.container = "super cool blob container name"
 azure.blob.account = "my awesome azure account name"
 
+auth.applicant_idp = "disabled"
```

Comment:
Not blocking: Was this change necessary?

Reply:
It was just lower down and I moved it up.